lib/capybara/selenium/driver.rb

```diff
@@ -148,8 +148,12 @@ def evaluate_async_script(script, *args)
     unwrap_script_result(result)
   end
 
+  def active_element
+    build_node(browser.switch_to.active_element)
+  end
+
   def send_keys(*args)
-    active_element.send_keys(*args)
+    active_element.native.send_keys(*args)
```

Comment:
I made this change before https://github.com/teamcapybara/capybara/pull/2489/files#r689119285.

Should we revert it so that the `send_keys` goes through the specialized nodes?